.github/actions/package-nfpm/action.yml

```diff
@@ -115,10 +118,12 @@ runs:
         key: ${{ inputs.cache_key }}
 
     # Update if condition to true to get packages as artifacts
-    - if: ${{ false }}
+    - if: |
+        contains(github.event.pull_request.labels.*.name, 'upload-artifacts') ||
+        inputs.is_nightly == true
```

Comment:
I think this condition will never match since composite action inputs are considered as string (not boolean)